pkgs/development/haskell-modules/configuration-ghc-9.8.x.nix

```diff
@@ -48,27 +49,119 @@ self: super: {
   unix = null;
   xhtml = null;
 
-  # https://github.com/tibbe/unordered-containers/issues/214
-  unordered-containers = dontCheck super.unordered-containers;
+  aeson = super.aeson_2_2_1_0;
+  aeson-pretty = super.aeson-pretty_0_8_10;
+  attoparsec-aeson = super.attoparsec-aeson_2_2_0_1;
+  alex = super.alex_3_4_0_1;
+  bifunctors = super.bifunctors_5_6_1;
+  cabal-install-solver = super.cabal-install-solver_3_10_2_1;
+  doctest = super.doctest_0_22_2;
+  free = super.free_5_2;
+  # ghc-lib 9.8.1.20231121 required for Cabal to build: https://github.com/digital-asset/ghc-lib/issues/495
+  ghc-lib = super.ghc-lib_9_8_1_20231121;
+  ghc-lib-parser = super.ghc-lib-parser_9_8_1_20231121;
+  ghc-lib-parser-ex = super.ghc-lib-parser-ex_9_8_0_0;
+  github = super.github_0_29;
+  hedgehog = super.hedgehog_1_4;
+  hspec = super.hspec_2_11_7;
+  hspec-core = super.hspec-core_2_11_7;
+  hspec-discover = super.hspec-discover_2_11_7;
+  hspec-hedgehog = super.hspec-hedgehog_0_1_1_0;
+  hspec-meta = super.hspec-meta_2_11_7;
+  megaparsec = super.megaparsec_9_6_1;
+  megaparsec-tests = super.megaparsec-tests_9_6_1;
+  ormolu = super.ormolu_0_7_3_0;
+  rebase = super.rebase_1_20_1_1;
+  rerebase = super.rerebase_1_20_1_1;
+  semigroupoids = super.semigroupoids_6_0_0_1;
+  singleton-bool = super.singleton-bool_0_1_7;
+  some = super.some_1_0_6;
+  tagged = super.tagged_0_8_8;
+  tasty-hspec = super.tasty-hspec_1_2_0_4;
+  th-abstraction = super.th-abstraction_0_6_0_0;
+
+  cabal-fmt =
+    if pkgs.stdenv.targetPlatform.isDarwin && pkgs.stdenv.targetPlatform.isAarch64 then
+      overrideCabal (drv: {
+        enableSeparateBinOutput = false;
+      }) (super.cabal-fmt)
+    else
+      super.cabal-fmt;
+
+  ChasingBottoms = dontCheck (doJailbreak super.ChasingBottoms); # base >=4.2 && <4.19
+
+  # https://github.com/obsidiansystems/commutative-semigroups/issues/13
+  commutative-semigroups = doJailbreak super.commutative-semigroups; # base >=4.6 && <4.19
+
+  dates = doJailbreak super.dates; # base >=4.9 && <4.16
+
+  double-conversion = overrideCabal (drv: {
+    src = pkgs.fetchFromGitHub {
+      owner = "haskell";
+      repo = "double-conversion";
+      rev = "d480fb057c5387251b8cfdeb3666b24087811219";
+      sha256 = "1f2q5p9nh4ccyrkc2fx6xjiq8v15i6myf01ajcw541dqc8z5aiw0";
+    };
+    editedCabalFile = null;
+    license = pkgs.lib.licenses.bsd2;
+  }) super.double-conversion;
+
+  # Tests fail due to the newly-built fourmolu not being in PATH
```

Comment:
You can allow this by adding an additional override like this for `fourmolu_0_14_1_0` in `configuration-nix.nix`:

https://github.com/NixOS/nixpkgs/blob/55ca7939944136731b34316a4c6fc2d4a11da122/pkgs/development/haskell-modules/configuration-nix.nix#L1195-L1200

Unfortunately the override isn't applied to the versioned attribute due to overlay ordering.